README.md

```diff
@@ -42,11 +42,34 @@ mkdir -p /var/tmp/reva/root/{home,oc}
 
 ## Quickstart
 
-After building the binary with the above steps and also setting up the prerequisites, run this command to start all services:
-
-```console
-./bin/ocis server
-```
+After building the binary with the above steps and setting up the prerequisites
+
+1. create a file called `identifier-registration.yml` with this content:
+
+    ```yml
+     ---
+
+     # OpenID Connect client registry.
+     clients:
+        - id: phoenix
+          name: ownCloud web
+          application_type: web
+          insecure: yes
+          trusted: yes
+          redirect_uris:
+            - https://localhost:9200/oidc-callback.html
+            - https://localhost:9200/
+          origins:
+            - https://localhost:9200
+
+      authorities:
+    ```
+
+2. run this command to start all services:
+
+    ```console
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-content-indent] Don’t use mixed indentation for children, remove 1 space](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=5393452)